src/Templates/NIPSWeb/manage_library.twig

```diff
@@ -1,30 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Should the associated controller be deleted as well?

Reply:
I forget, if it's not the same as the manual upload training one, then yes.

Reply:
Doesn't look like it's actually in the code anymore, just the manual upload one is in there.